src/speech.cs

```diff
@@ -241,6 +259,38 @@ interface SFTranscriptionSegment : NSCopying, NSSecureCoding {
 
 		[Export ("alternativeSubstrings")]
 		string [] AlternativeSubstrings { get; }
+
+		[iOS (13, 0)]
+		[NullAllowed, Export ("voiceAnalytics")]
+		SFVoiceAnalytics VoiceAnalytics { get; }
+	}
+
+	[Mac (10,15, onlyOn64: true), iOS (13,0)]
+	[BaseType (typeof (NSObject))]
+	interface SFAcousticFeature : NSCopying, NSSecureCoding {
+
+		[Export ("acousticFeatureValuePerFrame", ArgumentSemantic.Copy)]
+		NSNumber[] AcousticFeatureValuePerFrame { get; }
```

Comment:
check documentation - it might be possible to bind something better (more accurate) than `NSNumber[]`, e.g. `float[]`

Reply:
So all the documentation says is that it refers to a feature value which I assume will be a decimal number, but how would one decide if a float, double, or decimal would be more appropriate?

Reply:
Not a `decimal` for sure - its more costly to work with :)

The main _problem_ with exposing `NSNumber` is that someone might not use the _best_ type... and it force them to convert them to native C# types.

The first one is not a big deal here since it's a getter (it's only read, not user supplied). Since the current documentation is not clear then I rather err on the safe side (and keep `NSNumber`). @dalexsoto any preference on your side ?

Reply:
Agreed, let's keep `NSNumber` for now, maybe we can also check the swift side https://developer.apple.com/documentation/speech/sfacousticfeature/3044203-acousticfeaturevalueperframe in the future just in case they provide us a hint on the type they are actually boxing.

Reply:
Thanks! Will do!